lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -218,5 +218,6 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
         "transcripts": transcripts,
         "language": video_descriptor.get_default_transcript_language(),
         "category": video_descriptor.category,
+        "only_on_web" : video_descriptor.unavailable_on_mobile,
```

Comment:
Talked to @nasthagiri a bit about it after a clarification from Lou. The conclusion was that we're going to make the property ``only_on_web``, but not worry about syncing it up the ``mobile_available`` flag.